build-farm/set-platform-specific-configurations.sh

```diff
@@ -25,5 +25,44 @@ fi
 
 export VARIANT_ARG="--build-variant ${VARIANT}"
 
-# shellcheck disable=SC1091,SC1090
-source "$SCRIPT_DIR/platform-specific-configurations/${OPERATING_SYSTEM}.sh"
\ No newline at end of file
+# Create placeholders for curl --output
```

Comment:
Could some of this script be refactored into functions?

Reply:
Done